gateway-server/src/main/java/org/apache/knox/gateway/GatewayServer.java

```diff
@@ -923,6 +925,40 @@ private synchronized void internalActivateArchive( Topology topology, File warDi
     }
   }
 
+  private synchronized void activateHomeApp() {
+    try {
+      // UI
+      activateHomepageContext("/home", "homepage/%2Fhome");
+
+      // API
+      activateHomepageContext("/home/api", "homepage/%2F");
+    } catch (Throwable e) {
+      e.printStackTrace();
```

Comment:
remove `e.printStackTrace()`

why are we catching `Throwable` that seems bad

Reply:
Deleted.